make/modules/jdk.jfr/Java.gmk

```diff
@@ -23,6 +23,6 @@
 # questions.
 #
 
-DISABLED_WARNINGS_java += exports
+DISABLED_WARNINGS_java += exports lossy-conversions
```

Comment:
Note that with the fix of JDK-8286392 (and JDK-8286396) the `lossy-conversions` warning should not be disabled for the JFR code. 

In general, you need to check which of the subtasks of JDK-8286374 that has been fixed, and adjust the makefiles accordingly, before pushing this fix.

(In the future, it might be easier to push the fix which disables the warnings first, and then file follow-up bugs on aa per-component basis, and remind them to remove the disabling in the makefile. That way there won't be a race between individual fixes and a "master" bug like this.)

Reply:
That's good to know. I think the tricky part is mostly about keeping track of all these disabled warnings, so they are not kept around longer than necessary. And that needs coordination with all the subtasks of the umbrella issue.

Reply:
I put out a PR for java.base, but thought I'd wait until the javac fixe were pushed before integrating and would re-enable the warnings at the same time.

Reply:
Feel free to go ahead with the java.base PR as this one still needs CSR.